core/sagas/blueprints.js

```diff
@@ -210,6 +211,36 @@ function* setBlueprintUsers(action) {
   }
 }
 
+// convert between our stores blueprint object and the format that the weldr api expects
+function createWeldrBlueprint(blueprint) {
+  const weldrBlueprint = {
+    name: blueprint.name,
+    description: blueprint.description,
+    version: blueprint.version,
+    modules: blueprint.modules,
+    packages: blueprint.packages,
+    groups: blueprint.groups !== undefined ? blueprint.groups : [],
+    customizations: blueprint.customizations,
+  };
+  return weldrBlueprint;
+}
+
+function* setBlueprintDevice(action) {
+  try {
+    const { blueprint, device } = action.payload;
+    // commit the oldest blueprint with the updated hostname
```

Comment:
Comment isn't entirely correct here I think?

Reply:
Thanks. Forgot to remove this comment.